ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -873,5 +873,68 @@
     "CaseEventID": "manageRepresentatives-PREPARE_FOR_HEARING",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
+  },
```

Comment:
do other case roles need to see that this event has happened?

Reply:
Good spot! They will need to